prboom2/src/MUSIC/flplayer.c

```diff
@@ -105,9 +107,51 @@ static const char *fl_name (void)
 #include <delayimp.h>
 #endif
 
+static void *fl_sfopen(const char *lumpname)
```

Comment:
Let's pass lumpnum instead of lumpname here and save a lookup.

Reply:
This is a callback, so we need to pass `const char *`. We can save an int to a string, or convert pointers, or make `lumpnum` global, but I don't think it's worth it.

Reply:
Ah ok, then all good 👍